tensorboard/plugins/scalar/vz_line_chart/vz-line-chart.ts

```diff
@@ -244,7 +289,12 @@ Polymer({
     if (this._chart) {
       this._chart.setTooltipPosition(this.tooltipPosition);
     }
-  }
+  },
+  _computeTooltipTableHeaderHtml(tooltipColumns) {
+    // The first column contains the circle with the color of the run.
+    const mapper = (c: ChartHelpers.TooltipColumn) => `<th>${c.title}</th>`;
+    return '<th></th>' + _.map(tooltipColumns, mapper).join('');
```

Comment:
Another way to write this is
```js
const titles = ["", ..._.map(tooltipColumns, 'title')];
return titles.map(title => `<th>${sanitize(title)}</th>`).join('');
```
I'm slightly partial to it—I think it's a bit more direct—but the choice is yours.

(Plus, if you're using `_.map` over `Array.prototype.map`, at least take advantage of its plucking feature!)